packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -658,6 +661,13 @@ export class RemoteMediaManager extends EventsScope {
       this.slots.video.activeSpeaker.length +
       this.slots.video.receiverSelected.length;
 
+    // move all no longer needed receiver-selected slots to "unused"
+    this.trimReceiverSelectedSlots();
```

Comment:
this function adds slots that were previously receiver-selected to "unused" - we probably would want these to be after all the previous "active speaker" slots, not before. This is not an issue right now, but will be when we have stage layout. I think we just need to move this line of code to be after the code from the lines 668-669 or maybe even after the block that allocates fresh new slots?

Reply:
yeah, makes sense.
I've refactored this quite a bit, hope it makes more sense now.